src/Pages/Home/Home.jsx

```diff
@@ -1,49 +1,50 @@
 import "../../css/main.css";
 import "./home.css";
-import { DisplayCard, InputCard } from "../../Components";
-import { Header, Footer } from "../../Components";
+import { useState } from "react";
+import { DisplayCard, InputCard, DisplayCardEmpty, Header, Footer, Sidebar } from "../../Components";
+import { useAuth } from "../../Context";
 
 export const Home = () => {
+    const [createNewCard, setCreateNewCard] = useState(false)
+    const { authState } = useAuth();
+    const { notes } = authState;
+
+    const pinnedNotes = notes.filter((item) => item.pinned)
+    const otherNotes = notes.filter((item) => !item.pinned)
```

Comment:
suggestion: costly operation runs on every render, put filters inside useEffect with dependcy array

Reply:
yes thanks for suggesting this. Will change it